examples/isolated-objects/isolated_objects.bal

```diff
@@ -0,0 +1,86 @@
+import ballerina/io;
+
+type Coordinates record {|
+    decimal latitude;
+    decimal longitude;
+|};
+
+isolated class Cities {
+
+    final string country;
+
+    // The initialization expression of a field of an
+    // `isolated` object has to be an `isolated` expression,
+    // which itself will be an isolated root.
+    // e.g., expressions with an immutable static type,
+    // simple literals, constructor expressions
+    // where all the subexpressions are isolated expressions, etc.
+    private map<Coordinates> cities = {};
+
+    function init(string country) {
+        self.country = country;
+    }
+
+    function add(string city, Coordinates coordinates) {
+        lock {
+            // Where `self` is being used to access a mutable private
```

Comment:
```suggestion
            // Where `self` is being used to access a mutable, private
```

Reply:
It is the private field that is mutable. Do we need commas?